tests/SideBySide.New/ConnectionPool.cs

```diff
@@ -148,9 +118,5 @@ public void ResetConnection(object connectionReset, int poolSize, long expected)
 				connection.Dispose();
 		}
 
-		public void Dispose()
-		{
-			MySqlConnection.ClearAllPools();
```

Comment:
I agree that clearing pools at the end of each test isn't ideal, but I'm not clear on how this "wreaks havoc"; are there tests that specifically rely on idle connections being available? 

This may actually indicate a bug in `ClearAllPools`; it shouldn't be implemented as a "destructive" operation for existing open connections but as per the [documentation](https://dev.mysql.com/doc/dev/connector-net/html/M_MySql_Data_MySqlClient_MySqlConnection_ClearPool.htm)

> ClearPool clears the connection pool that is associated with the connection. If additional connections associated with connection are in use at the time of the call, they are marked appropriately and are discarded (instead of being returned to the pool) when Close is called on them.

(I don't think my implementation guarantees that, so I'll need to fix that up.)